src/impl_/pymodule.rs

```diff
@@ -132,6 +132,17 @@ impl ModuleDef {
     }
 }
 
+/// Trait to add an element (class, function...) to a module
+pub trait PyAddToModule {
+    fn add_to_module(module: &PyModule) -> PyResult<()>;
+}
+
+impl<T: PyTypeInfo> PyAddToModule for T {
+    fn add_to_module(module: &PyModule) -> PyResult<()> {
+        module.add(Self::NAME, Self::type_object(module.py()))
```

Comment:
`add_class` uses `T::lazy_type_object().get_or_try_init(py)?` instead of `T::type_object(py)`. Is there a reason to prefer one or the other?